packages/firestore-types/index.d.ts

```diff
@@ -57,6 +57,22 @@ export interface Settings {
   timestampsInSnapshots?: boolean;
 }
 
+/** Settings used to configure Firestore persistence. */
+export interface PersistenceSettings {
+  /**
+   * Whether to synchronize the in-memory state of multiple tabs. Setting this
+   * to 'true' in all open tabs enables shared access to local persistence,
+   * shared execution of queries and latency-compensated local document updates
+   * across all connected instances.
+   *
+   * To enable this mode, `synchronizeTabs:true` needs to be set globally in
+   * all active tabs.
+   *
+   * NOTE: This mode is experimental.
```

Comment:
FWIW- This isn't super actionable.  Consider adding "and not recommended for production use" or "and may cause bugs" or "and may corrupt your offline cache" or "Please log bugs on [github issues link] if you see any issues" or something (depending on how cautious we want people to be and what we want them to do).

Reply:
I made it `not yet recommended for production use`.